packages/ui-components/src/lib/components/detail/OrderDetail.svelte

```diff
@@ -100,6 +100,41 @@
 		}
 	};
 
+	$: dotrainSource = $orderDetailQuery.data?.parsedMeta?.find(
+		(meta) => 'DotrainSourceV1' in meta
```

Comment:
why do have to do this in the svelte component? this should be in the sdk - we can still include all meta found, sure, but DotrainSourceV1 and DotrainGuiStateV1 are important enough they should just have their own place on the order object and be ready for use immediately